File: ncDados/dEfdEnderecoContato.vb
Namespace nsEFD

  Public Class dEfdEnderecoContato

    Private _logradouro As String
    Private _numero As String
    Private _complemento As String
    Private _bairro As String
    Private _cep As String
    Private _dddTelefone As String
    Private _dddFax As String
    Private _email As String

    Public Property logradouro() As String
      Get
        Return _logradouro
      End Get
      Set(ByVal value As String)
        _logradouro = value
      End Set
    End Property

    Public Property numero() As String
      Get
        Return _numero
      End Get
      Set(ByVal value As String)
        _numero = value
      End Set
    End Property

    Public Property complemento() As String
      Get
        Return _complemento
      End Get
      Set(ByVal value As String)
        _complemento = value
      End Set
    End Property

    Public Property bairro() As String
      Get
        Return _bairro
      End Get
      Set(ByVal value As String)
        _bairro = value
      End Set
    End Property

    Public Property cep() As String
      Get
        Return _cep
      End Get
      Set(ByVal value As String)
        _cep = value
      End Set
    End Property

    Public Property dddTelefone() As String
      Get
        Return _dddTelefone
      End Get
      Set(ByVal value As String)
        _dddTelefone = value
      End Set
    End Property

    Public Property dddFax() As String
      Get
        Return _dddFax
      End Get
      Set(ByVal value As String)
        _dddFax = value
      End Set
    End Property

    Public Property email() As String
      Get
        Return _email
      End Get
      Set(ByVal value As String)
        _email = value
      End Set
    End Property

  End Class

End Namespace

File: ncModelos/dEfdEnderecoContato.cs
using System;

namespace ncModelos
{
    public class dEfdEnderecoContato
    {
        private string _logradouro;
        private string _numero;
        private string _complemento;
        private string _bairro;
        private string _cep;
        private string _dddTelefone;
        private string _dddFax;
        private string _email;

        public string logradouro
        {
            get => _logradouro;
            set => _logradouro = value;
        }

        public string numero
        {
            get => _numero;
            set => _numero = value;
        }

        public string complemento
        {
            get => _complemento;
            set => _complemento = value;
        }

        public string bairro
        {
            get => _bairro;
            set => _bairro = value;
        }

        public string cep
        {
            get => _cep;
            set => _cep = value;
        }

        public string dddTelefone
        {
            get => _dddTelefone;
            set => _dddTelefone = value;
        }

        public string dddFax
        {
            get => _dddFax;
            set => _dddFax = value;
        }

        public string email
        {
            get => _email;
            set => _email = value;
        }
    }

}
